Deepgram/LiveClient.cs

```diff
@@ -9,11 +9,9 @@ public class LiveClient : IDisposable
 {
     public LiveClient(string apiKey, DeepgramClientOptions? deepgramClientOptions = null)
     {
+        if (string.IsNullOrEmpty(apiKey)) throw new ArgumentNullException(nameof(apiKey), "ApiKey is required");

```

Comment:
This is going to sound odd (trust me), but in the case of OnPrem, the API Key is empty and we need to allow for it.